src/components/AppCatalog/AppDetail/InstallAppModalUtils.ts

```diff
@@ -0,0 +1,42 @@
+const maxLength = 253;
```

Comment:
This was `63` before, even though the error message stated `253`

Reply:
Im not sure which is right though... the error message or the validation. I think the 63 has a purpose, because the name of the app get added to something else which then might get longer than 253